src/examples/05-custom-dialog.html

```diff
@@ -1,89 +0,0 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
@winhamwr I've removed this just before merging now.

This would be a bad example as it is.

The API seems well documented, so an example seems not so important.